src/charm.pyi

```diff
@@ -23,4 +24,31 @@ class WordpressCharm(ops.charm.CharmBase):
             event: opslib.mysql.MySQLDatabaseChangedEvent
     ): ...
 
-    def _gen_wp_config(self) -> str: ...
\ No newline at end of file
+    def _gen_wp_config(self) -> str: ...
+
+    def _container(self) -> ops.charm.model.Container: ...
+
+    def _wordpress_service_exists(self) -> bool: ...
+
+    def _stop_server(self): ...
```

Comment:
Should the return value here be `None`? Same comment applies to any other functions here without a return type

Reply:
I think they are none by default? But I will add them for less confusion.